test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -2217,9 +2212,8 @@ func Test_CostLimitForValidation(t *testing.T) {
 					Name: "test-k8s",
 				},
 			},
-			err:                   "operation cancelled: actual cost limit exceeded",
-			failureReason:         metav1.StatusReasonInvalid,
-			strictCostEnforcement: true,
+			err:           "operation cancelled: actual cost limit exceeded",
+			failureReason: metav1.StatusReasonInvalid,
 		},
 		{
 			name:          "With StrictCostEnforcementForVAP: Expression exceeds per policy cost limit for extended library",
```

Comment:
this test is failing currently.

Reply:
is this the failure?

>  resp=422 validatingadmissionpolicy_test.go:3272: actual error: namespaces "test-k8s" is forbidden: ValidatingAdmissionPolicy 'validate-namespace-suffix' with binding 'validate-namespace-suffix-binding' denied request: expression '[1, 2, 3, 4, 5, 6, 7, 8, 9, 10].all(x, [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].all(y, [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].all(z, [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].all(z2, [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].all(z3, [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].all(z4, [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].all(z5, int('1'.find('[0-9]*')) < 100)))))))' resulted in error: operation cancelled: actual cost limit exceeded validatingadmissionpolicy_test.go:3273: expected error: validatingadmissionpolicy_test.go:3274: got error but expected none ---

Reply:
I guess.. ? Overall, it was the same issue affecting all previously failing tests i.e. the policies not being refreshed soon enough, and so the old policies were still being evaluated against new testcases that were using different policies.